### ISSUE TITLE
False error when JSDoc comment line length is exactly 80 chars

### ISSUE THREAD

**Issue by jaydenseric** [lasselupe33/eslint-plugin-comment-length#6]
I configure VS Code with a vertical ruler at 80 chars:

```json
{
  "editor.rulers": [80]
}
```

As you can see, there is a false rule `comment-length/limit-multi-line-comments` error when the JSDoc comment line length is exactly 80 chars:

<img width="745" alt="Screenshot 2023-07-04 at 7 24 15 am" src="https://github.com/lasselupe33/eslint-plugin-comment-length/assets/1754873/9a463451-0f96-4d1e-bb9d-74404ae643fd">

I didn't see a false error like this until updating the ESLint plugin recently.

**Comment by lasselupe33** [comment]
Hi @jaydenseric,

You're absolutely correct. A regression was introduced in the latest release. This off-by-one error has been fixed in **v1.4.5**.
